data/campaigns/Delfadors_Memoirs/scenarios/06_Swamps_of_Illuven.cfg

```diff
@@ -169,11 +210,222 @@
+                                                            [/modify_side]
+                                                            {VARIABLE garrath_hostile yes}
+                                                        [/command]
+                                                    [/option]
+                                                [/message]
+                                            [/else]
+                                        [/if] # gold
+                                        {CLEAR_VARIABLE gold}
+                                    [/then]
+                                    [else]
+                                        [message]
+                                            speaker=Garrath
+                                            message=_"Stop stealing my villages!"
+                                        [/message]
+                                    [/else]
+                                [/if] # garrath_hostile
+                            [/then]
+                        [/if]
+                    [/event]
+
+                    [event]
+                        name=side 1 turn refresh
+                        first_time_only=no
+                        [store_gold]
+                            variable=gold
+                            side=1
+                        [/store_gold]
+                        [if]
+                            [variable]
+                                name=gold
+                                less_than=0
+                            [/variable]
+                            [then]
+                                [message]
+                                    speaker=Garrath
+                                    message=_"Hm, it looks like you don't have any gold left. We're not interested in you anymore; bye!"
+                                [/message]
+                                [kill]
+                                    side=2
+                                [/kill]
+                                [message]
+                                    speaker=Harold
+                                    message=_"What a fickle one, that Garrath..."
+                                [/message]
+                            [/then]
+                        [/if]
+                        {CLEAR_VARIABLE gold}
+                    [/event]
+
+                    # A little bit of extra balance to make up for the fact that Harold now has an extra enemy:
+                    [event]
+                        name=last breath
+                        [filter]
+                            side=2
+                        [/filter]
+                        [filter_second]
+                            side=3
+                        [/filter_second]
+                        [message]
+                            speaker=Harold
+                            message=_"I'll be taking back that money that your men ""borrowed"" from me now, Garrath!"
```

Comment:
For this line, instead of using "", use a unicode left and right double quote.

Reply:
> For this line, instead of using "", use a unicode left and right double quote.

ok, done in d242e6a